src/terminal.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const readline = require('readline');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+exports.terminal = terminal;
```

Comment:
Instead of exposing the entire readline inteface, export only the methods you need for this app (question, log, etc.)